frontend/talentsearch/src/js/directIntakeRoutes.ts

```diff
@@ -22,6 +22,66 @@ const directIntakeRoutes = (lang: string) => {
       path.join(home(), "applications", id, "submit"),
     applications: (userId: string) =>
       path.join(userHome(userId), "applications"),
+    aboutMe: (poolCandidateId: string) =>
+      path.join(home(), "applications", poolCandidateId, "apply", "about-me"),
```

Comment:
This seems like a strange approach to me - to create all new routes instead of reusing the ones that already exist in profileRoutes.ts.  Reusing the existing routes is why we wanted to add the extra applicationId option to the URL, I think.  We want to avoid having multiple routes that go to the same page.

Reply:
Fixed by [Refactor jobtitle and navigational messages in review application page and edit pages](https://github.com/GCTC-NTGC/gc-digital-talent/pull/3796/commits/6284d286bc06ad582b51d5da87f5d4f8184c308d)